hilti/include/ast/expressions/unresolved-operator.h

```diff
@@ -25,9 +25,9 @@ class UnresolvedOperator : public NodeBase, public trait::isExpression {
     }
 
     /** Implements `Expression` interface. */
```

Comment:
Nice, now almost all methods in this block are dummy-impls. Let's add a comment 
```suggestion
    /** Implements `Expression` interface. */
    
    // Dummy implementations as the node will be rejected in validation anyway.
```

and an empty line before the `isEqual` impl (could also remove l.37 so dummy and real impls are in blocks, each).